aiohttp/client_reqrep.py

```diff
@@ -500,6 +505,7 @@ class ClientResponse(HeadersMixin):
     raw_headers = None  # Response raw headers, a sequence of pairs
 
     _connection = None  # current connection
+    _content_disposition = False  # content disposition cache
```

Comment:
I still think that's not good choice. `None` means no value. We tried to parse it, nothing happened. Yes, this will trigger headers lookup for each attribute access - but that's not a problem.

Reply:
`helpers.sentinel` is an option

Reply:
Good alternative, but `None` is fine as well. However, even better would be to use reify here and avoid internal attributes. Headers are immutable in anyway.

Reply:
+1 for `reify`